docs/Components/messages.md

```diff
@@ -17,11 +17,13 @@ property object with the following properties:
 | Property | Description  | Type | Required |
 | -- | -- | --| -- |
 | id | A unique identifier for the message | String | Yes |
+| usesSharedStrings | Whether it uses the translation file shared between all messages (default: false) | Boolean | No |
+| shortVersion | A human-readable version number to insert into user-facing strings | Strings | Yes if usesSharedStrings is true, No otherwise |
```

Comment:
nit: This PR is a lot of work to go through, and it leaves us only one variable which can be set in common strings.

Please note, I am not asking to change this... more just pondering about the extensibility of this feature.

Reply:
I started to draft out a generic variable system in these strings, but quickly realized I could not dream up a use case where we'd need it beyond "drop in the version". We wouldn't have a variable that is number (thus needing singular/plural) in a message, I don't think. In general, we never have variables in messages even. 

If you can find an old message where this would be helpful, or have a potential use case, let me know and I'm happy to include it.